src/modules/partition/jobs/SetPartitionFlagsJob.cpp

```diff
@@ -62,38 +62,38 @@ SetPartFlagsJob::prettyDescription() const
     {
         if ( !partition()->partitionPath().isEmpty() )
         {
-            return tr( "Clear flags on partition <strong>%1</strong>." ).arg( partition()->partitionPath() );
+            return tr( "Clearing flags on partition <strong>%1</strong>…", "@status" ).arg( partition()->partitionPath() );
         }
 
         QString fsNameForUser = userVisibleFS( partition()->fileSystem() );
         if ( !fsNameForUser.isEmpty() )
         {
-            return tr( "Clear flags on %1MiB <strong>%2</strong> partition." )
+            return tr( "Clearing flags on %1MiB <strong>%2</strong> partition…", "@status" )
                 .arg( BytesToMiB( partition()->capacity() ) )
                 .arg( fsNameForUser );
         }
-        return tr( "Clear flags on new partition." );
+        return tr( "Clearing flags on new partition…", "@status" );
     }
 
     if ( !partition()->partitionPath().isEmpty() )
     {
-        return tr( "Flag partition <strong>%1</strong> as "
-                   "<strong>%2</strong>." )
+        return tr( "Flagging partition <strong>%1</strong> as "
```

Comment:
"Flagging" seems like a strange word to use in this context.

If we want to change it like that perhaps it should be `Setting flags on partition"

Reply:
This string needs to be taken in context, it is in the `prettyDescription()` function, so this is a longer human-readable description of what the job will do during installation. So "flagging" is not right. "Set flag %2 .." is better and matches the "clear flag" description it can also give. The job name is "Set flags on partition" and the status messages are "Clearing ..." and "Setting ..."